wire-ios-request-strategy/Sources/Helpers/EncryptionSessionDirectory+UpdateEvents.swift

```diff
@@ -82,8 +83,8 @@ extension EncryptionSessionsDirectory {
 
     /// Appends a system message for a failed decryption
     fileprivate func appendFailedToDecryptMessage(after error: CBoxResult?, for event: ZMUpdateEvent, sender: UserClient, in moc: NSManagedObjectContext) {
-        zmLog.safePublic("Failed to decrypt message with error: \(error), client id <\(sender.safeRemoteIdentifier))>")
-        zmLog.error("event debug: \(event.debugInformation)")
+        logger.debug("Event debug: \(event.debugInformation)")
```

Comment:
This log doesn't add any new information that the next log already adds, so I would remove it.

Reply:
removed additional log